apps/passport-server/src/database/queries/rateLimit.ts

```diff
@@ -0,0 +1,20 @@
+import { Pool } from "postgres-pool";
+import { RateLimit } from "../models";
+import { sqlQuery } from "../sqlQuery";
+
+export async function checkRateLimit(
```

Comment:
suggestion: I think adding a comment here would be helpful. in the comment it may be helpful to link to the [original implementation](https://github.com/fafl/token-bucket-postgres), and explain some of the caveats / limitations.